topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: tutorial_hands_on
+topic_name: epigenetics
+tutorial_name: hicexplorer
+---
+
+# Introduction
+{:.no_toc}
+
+In this HiCExplorer tutorial we will generate and plot a Hi-C contact matrix.
+For this the following steps are necessary to be performed:
+1. Map the Hi-C reads to the reference genome
+2. Creation of a Hi-C matrix
+3. Plotting the Hi-C matrix
+4. Correction of Hi-C matrix
+5. TAD Calling
+
+After a corrected Hi-C matrix is created other tools can be used to visualize it, call TADS or compare it with other matrices.
+This tutorial is based on the [following work](http://hicexplorer.readthedocs.io/en/latest/content/example_usage.html).
+
+We will be using a Hi-C dataset from [Marks et. al. 2015](http://www.genomebiology.com/2015/16/1/149), on mouse ESCs.
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Data upload
+

```

Comment:
For the galaxy course, import the pre-processed matrices SRR1956527.h5, SRR1956528.h5  and SRR1956529.h5.

They will be on zenodo and in the Shared Library. (Shared Data -> Data Libraries ->  )

The alignment and the build step of the three contact matrices can than be skipped. It takes a few hours to compute it.